tests/covalent_dispatcher_tests/_cli/cli_test.py

```diff
@@ -41,7 +41,6 @@ def test_cli(mocker):
         ("python" in response.output.lower())
         and ("agnostiq" in response.output.lower())
         and ("copyright" in response.output.lower())
-        and (current_version in response.output.lower())
```

Comment:
do we not want the version in the output?

Reply:
The actual version in the branch is ahead (0.158.0) vs the version in the CLI --version command (Release version 0.156.0). So, I figured this test logic is not quite right.